cursorless-nx/package.json

```
@@ -45,7 +45,7 @@
 		"react-test-renderer": "18.1.0",
 		"ts-jest": "27.1.4",
 		"ts-node": "10.8.2",
-		"typescript": "4.7.4",
+		"typescript": "4.8.3",
```

Probably a good idea to pin the minor version instead of the patch version.
```suggestion
		"typescript": "~4.8.3",
```
Although I guess I can make the same note about all the version pinning happening in this file 😕